bigquery_etl/backfill/shredder_mitigation.py

```diff
@@ -50,30 +67,187 @@ class DataTypeGroup(Enum):
     UNDETERMINED = "None"
 
 
+@attr.define
 class Column:
     """Representation of a column in a query, with relevant details for shredder mitigation."""
 
-    def __init__(self, name, data_type, column_type, status):
-        """Initialize class with required attributes."""
-        self.name = name
-        self.data_type = data_type
-        self.column_type = column_type
-        self.status = status
-
-    def __eq__(self, other):
```

Comment:
why did you get rid of the __eq__? that sounds useful

Reply:
Cause I'm be changing it for an autogenerated `@attr.define(eq=True)`